lib/pages/login-page.js

```diff
@@ -96,19 +95,6 @@ export default class LoginPage extends AsyncBaseContainer {
 	}
 
 	static getLoginURL() {
-		const baseURL = dataHelper.configGet( 'calypsoBaseURL' );
-		let loginURL = `${ baseURL }/log-in`;
-
-		// If Calypso is running locally in Docker, force login to actual WordPress.com first
```

Comment:
we don't support local calypso docker any more - use calypso.live instead (which uses docker anyway)